networks/_modules/docker-besu/besu.tf

```diff
@@ -84,7 +84,6 @@ exec /opt/besu/bin/besu \
         --genesis-file=${local.container_besu_datadir}/genesis.json \
         --node-private-key-file=${local.container_besu_datadir}/key \
         --min-gas-price=0 \
```

Comment:
@pinges the Besu AT are failing with:
```
picocli.CommandLine$ParameterException: Cannot use GoQuorum genesis file without GoQuorum privacy enabled
STDERR: at org.hyperledger.besu.cli.BesuCommand.run(BesuCommand.java:1207)
STDERR: at picocli.CommandLine.executeUserObject(CommandLine.java:1939)
STDERR: at picocli.CommandLine.access$1300(CommandLine.java:145)
```

also, there is a warning: `main | WARN  | Besu | --min-gas-price ignored because none of --miner-enabled or --goquorum-compatibility-enabled was defined.`

could you check?

Reply:
There was an egg/hen problem on what to update first. Got the change into Besu first! The tests should succeed now.